TheWarWithin/RogueAssassination.lua

```diff
@@ -1428,18 +1331,16 @@ spec:RegisterAuras( {
     rupture = {
         id = 1943,
         duration = function () return 4 * ( 1 + effective_combo_points ) end,
-        tick_time = function () return ( debuff.rupture.exsanguinated and 2 or 1 ) * haste end,
+        tick_time = haste,
```

Comment:
Looks like `tick_time` should be `function() return 2 * haste end`; Rupture ticks every 2 seconds (but is hasted).